proofs/pfedit.ml

```diff
@@ -173,7 +173,7 @@ let build_by_tactic ?(side_eff=true) env sigma ?(poly=false) typ tac =
       const_entry_body = Future.chain ce.const_entry_body
         (fun (pt, _) -> pt, ()) } in
   let (cb, ctx), () = Future.force ce.const_entry_body in
-  let univs = UState.merge side_eff Evd.univ_rigid univs ctx in
+  let univs = UState.merge ~sideff:side_eff ~extend:status Evd.univ_rigid univs ctx in
```

Comment:
I don't think that's the right value for `extend`, it should probably be `true`.
Isn't the `status` about using `admit`?

Reply:
I have no idea, setting to true for now.